recipe/meta.yaml

```diff
@@ -17,14 +17,14 @@ package:
 source:
   - folder: {{ pkg_src }}
     url: https://{{ goname }}/archive/{{ version }}.tar.gz
-    sha256: d03208fd7ef983e4a606f861a5efcdc5a1980952a91febba3f67057800da6701
+    sha256: 1bef92674a09a9f35efa9ef14ee06c01101e586266641f6ffb5613cd726db143
 
   - folder: {{ runc_pkg_src }}
     url: https://{{ runc_goname }}/archive/v{{ runc_version }}.tar.gz
     sha256: 770e180d1cca1f3c1a67512ebf9720caafe97af4fc2fe12fa43a3cf7bfa77522
```

Comment:
We may also need [this `sys` package]( https://github.com/tianon/gosu/commit/c5f80cc5d2df0d626f4f470ecf4228512b2a1818#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R7 ). Going to try without it as it is listed as an indirect requirement.